performancebench/src/main/scala/cognite/spark/performancebench/Metrics.scala

```diff
@@ -0,0 +1,13 @@
+package cognite.spark.performancebench
+
+import io.prometheus.client.Summary
+
+object Metrics {
+  // We don't really know the buckets, so summary is a better metric type than histogram
```

Comment:
If we only run the tests once, maybe we can use a counter and measure the number of seconds each test takes? If we run them once every once in a while, a gauge describing the time it took for the last test run might be appropriate.

Reply:
Yes, so I've not decided yet if I want this as a cron job or as a deployment. So it depends a bit, one can specify the sliding window of summary as well.

Reply:
I went for the gauge method, so that we got more flexibility.